docs/plans/2026-03-10-paper-1-21-11-migration-plan.md

```
@@ -0,0 +1,272 @@
+### Task 2.3 — Spot-check 5 representative files
+
+Manually verify the replacement is correct in:
+- `src/main/kotlin/net/lumalyte/lg/interaction/menus/guild/GuildControlPanelMenu.kt`
+- `src/main/kotlin/net/lumalyte/lg/interaction/menus/management/ClaimFlagMenu.kt`
+- `src/main/kotlin/net/lumalyte/lg/utils/ClaimPermissionUtils.kt`
+- `src/main/kotlin/net/lumalyte/lg/utils/FlagUtils.kt`
+- `src/main/kotlin/net/lumalyte/lg/infrastructure/services/ToolItemServiceBukkit.kt`
+
+### Affected files (70 total)
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)